logdna/resource_view_test.go

```diff
@@ -503,6 +503,129 @@ func TestView_MultipleChannels(t *testing.T) {
+					testViewExists("logdna_view.test_view"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "name", "test"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "query", "test"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "apps.#", "2"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "apps.0", "app1"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "apps.1", "app2"),
+					resource.TestCheckResourceAttrPair(
+						"logdna_alert.test_preset_alert_ins",
+						"id",
+						"logdna_view.test_view",
+						"presetid",
+					),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "categories.#", "1"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "categories.0", "DemoCategory"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "hosts.#", "2"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "hosts.0", "host1"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "hosts.1", "host2"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "levels.#", "2"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "levels.0", "fatal"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "levels.1", "critical"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "tags.#", "2"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "tags.0", "tags1"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "tags.1", "tags2"),
+				),
+			},
+			{
+				Config: updCfg,
+				Check: resource.ComposeTestCheckFunc(
+					testViewExists("logdna_view.test_view"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "name", "test2"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "query", "query2"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "apps.0", "app3"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "apps.1", "app4"),
+					resource.TestCheckResourceAttrPair(
+						"logdna_alert.test_preset_alert_upd",
+						"id",
+						"logdna_view.test_view",
+						"presetid",
+					),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "categories.#", "1"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "categories.0", "DemoCategory"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "hosts.0", "host3"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "hosts.1", "host4"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "levels.0", "error"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "levels.1", "warning"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "tags.0", "tags3"),
+					resource.TestCheckResourceAttr("logdna_view.test_view", "tags.1", "tags4"),
+				),
+			},
+			{
+				ResourceName:      "logdna_view.test_view",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
```

Comment:
Not sure if you're doing anything different (compared to other tests), but this one was really easy to follow.  Good job on the names and flow.